hardhat.config.js

```diff
@@ -112,17 +112,16 @@ module.exports = {
       gas: 8500000, // Gas sent with each transaction (default: ~6700000)
       gasPrice: 20000000000, // 20 gwei (in wei) (default: 100 gwei)
       value: 20000000000,
-      accounts: [`${process.env.MNEMONIC}`]
+      accounts
     },
     harmony_test: {
       url: "https://api.s0.b.hmny.io",
       chainId: 1666700000,
       gas: 8500000, // Gas sent with each transaction (default: ~6700000)
       gasPrice: 20000000000, // 20 gwei (in wei) (default: 100 gwei)
       value: 20000000000,
-      accounts: [`${process.env.MNEMONIC}`]
+      accounts
     },
-
```

Comment:
Fixed these two things so tests would run. Not sure how you had hardhat working without it?